.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,22 +14,22 @@ https://github.com/obspy/obspy/wiki/ObsPy-Git-Branching-Model
 
 ### What does this PR do?
 
-*Please fill in*
+Fixes issue [#2543](https://github.com/obspy/obspy/issues/2543#issuecomment-580663921)
```

Comment:
You're not supposed to change the template _in your PR_; it's filled into the GitHub form when you open it and you're supposed to fill _that_ in.

Reply:
Understood; done.